developer_setup/plugins.md

```diff
@@ -36,6 +41,23 @@ git clone git@github.com:JoeSmith/manageiq-content.git plugins/manageiq-content
 git clone git@github.com:JoeSmith/manageiq-ui-classic.git plugins/manageiq-ui-classic
 ```
 
+#### Setting the correct remotes
+
+Inside your checkout you will have the default remote origin pointing to your fork. Now add an upstream remote which
+points to the upstream repository. To keep your fork up to date, you will have to merge changes from upstream
+into your fork.
+
+```bash
+cd manageiq-ui-classic
+git remote -v
+# origin  git@github.com:JoeSmith/manageiq-ui-classic.git (fetch)
+# origin  git@github.com:JoeSmith/manageiq-ui-classic.git (push)
+git remote add upstream git@github.com:ManageIQ/manageiq-ui-classic
+git fetch upstream
+# merge upstream into your fork
+git merge upstream/master && git push origin
```

Comment:
> Everyone tells me that they git push origin master and I'm not sure why that matters... do you know why?

Are you branching from upstream/master ? Or I dont understand what you mean...

> git push origin master

👍